src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -12,92 +12,187 @@ This page quickly describes how [Issues](https://guides.github.com/features/issu
 
 On GitHub, there are mainly three teams working together on the project.
 
-- Product Team: this team takes care of the product vision of the software. It has Product Managers, UX Designers and Wording Managers.
-- Quality Assurance (QA) team: this team takes care of ensuring the quality of the software. It has QA analysts and QA automation engineers.
-- Developers team a.k.a Core Team: this team takes care of the code and act as maintainer for the project.
+- Core Product Team: this team takes care of the product vision of the software. It has Product Managers, UX Designers, and a Product Content Manager (employees of the PrestaShop Company)
```

Comment:
Is it important to say who is an employee of the company and who isn't ? 🤔

Reply:
I think for clarity